gammapy/analysis/config.py

```diff
@@ -0,0 +1,189 @@
+from astropy.coordinates import Angle
+from astropy.time import Time
+from astropy.units import Quantity
+from gammapy.utils.scripts import make_path, read_yaml
+from pydantic import BaseModel, FilePath, validator
+from pydantic.utils import deep_update
+from pathlib import Path
+from typing import List
+from enum import Enum
+import yaml
+
+__all__ = ["Config", "General"]
```

Comment:
Remove "General" here? why expose it?

Maybe put "AnalysisConfig" instead of "Config"?
We might have other configs in Gammapy some day (e.g. for simulation)?
I'm 50:50 on this.

Reply:
Done.